objects/src/notes/note_id.rs

```diff
@@ -1,3 +1,5 @@
+use miden_crypto::utils::HexParseError;
+
 use super::{Digest, Felt, Hasher, Note, Word};
 use crate::utils::{
     serde::{ByteReader, ByteWriter, Deserializable, DeserializationError, Serializable},
```

Comment:
Can this not be imported from `crate::utils`?

Reply:
Done